st2common/st2common/models/db/execution.py

```diff
@@ -123,49 +122,39 @@ def get_masked_parameters(self):
         return serializable_dict['parameters']
 
 
-class ActionExecutionStdoutOutputDB(stormbase.StormFoundationDB):
+class ActionExecutionOutputDB(stormbase.StormFoundationDB):
     """
-    Stores stdout output of a particular action.
-
-    New document is inserted dynamically when a new line is received which means you can simulate
-    tail behavior by periodically reading from this collection.
-    """
-    execution_id = me.StringField(required=True)
-    action_ref = me.StringField(required=True)
-    timestamp = me.DateTimeField(required=True, default=date_utils.get_datetime_utc_now)
-
-    line = me.StringField()
-
-    meta = {
-        'indexes': [
-            {'fields': ['execution_id']},
-            {'fields': ['timestamp']},
-            {'fields': ['action_ref']}
-        ]
-    }
-
-
-class ActionExecutionStderrOutputDB(stormbase.StormFoundationDB):
-    """
-    Stores stderr output of a particular action.
-
-    New document is inserted dynamically when a new line is received which means you can simulate
-    tail behavior by periodically reading from this collection.
+    Stores output of a particular execution.
+
+    New document is inserted dynamically when a new chunk / line is received which means you can
+    simulate tail behavior by periodically reading from this collection.
+
+    Attribute:
+        execution_id: ID of the execution to which this output belongs.
+        action_ref: Parent action reference.
+        runner_ref: Parent action runner reference.
+        timestamp: Timestamp when this output has been produced / received.
+        type: Type of the output (e.g. stdout, stderr, output)
+        data: Actual output data. This could either be line, chunk or similar, depending on the
+              runner.
     """
     execution_id = me.StringField(required=True)
     action_ref = me.StringField(required=True)
+    runner_ref = me.StringField(required=True)
     timestamp = me.DateTimeField(required=True, default=date_utils.get_datetime_utc_now)
+    type = me.StringField(required=True, default='output')
```

Comment:
I am not a fan of using names of builtins as variable names

Reply:
I agree, will rename it to `output_type`.